packages/model/src/user.ts

```diff
@@ -199,3 +232,55 @@ export type FetchUsersFilter =
 export type FetchUsersOptions = Omit<FetchOptions<FetchUsersFilter>, 'search'>;
 
 export type UserRole = 'Staff' | 'Member' | 'None';
+
+export const toAlgoliaUserItem = (user: UserResponse): UserAlgoliaResponse => {
```

Comment:
we are not doing this for algolia - we are splitting the list response - period. That way both Algolia indexers and API responses need to be adjusted (you haven't done the latter).

Reply:
I've renamed it here [review: rename things with Algolia to UserItem](https://github.com/yldio/asap-hub/pull/4020/commits/22969e0ea8949b58dbba01630b2c9f591a51e206) + [fix rename mistake](https://github.com/yldio/asap-hub/pull/4020/commits/333767be894dce993026378c49ba9021305b93d9)